applications/registration-react/src/l10n/nb.json

```diff
@@ -331,6 +331,20 @@
     "loggedOutMsgPart2": "Sesjonen ble tidsavbrutt, men vi har tatt vare på alle endringene du har gjort. Logg inn på nytt for å fortsette å jobbe med registreringen."
   },
   "api": {
+    "import": {
```

Comment:
mangler nynorsk?

Reply:
Ja, det er begrenset støtte for språk i registrering, bare norsk og engelsk. Må komme som egen oppgave.